dyndxt_loader/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.18)
```

Comment:
Does it make sense to split `dyndxt_loader` into its own repo? This code doesn't seem nv2a-trace specific so it might be better to just import it as a module.

Reply:
It is its own repo (https://github.com/abaire/nxdk_dyndxt) and it is being pulled in as an `ExternalProject_Add` in the top level CMakeFileLists.txt.

This subdir is a hack to rebuild the dll_loader that is included in the dyndxt loader on the host so it can be used to bootstrap the loader (which needs to be relocated host-side).

At some point I'd expect this to be moved into xboxpy. Alternatively I guess we could also have the python wrapper either live in dyndxt_loader (which also doesn't seem exactly right to me, since the loader should be agnostic to how its injected, see https://github.com/abaire/xbdm_gdb_bridge/tree/main/src/handler_loader which is what I used to develop the loader) or a totally separate repo.

Reply:
I was referring to the loader (i.e. this directory), not dyndxt itself. Since it's independent of nv2a-trace I think it should be placed elsewhere, either into xboxpy or its own repo.
Perhaps xboxpy makes the most sense, as we could add logic to disable the xbdm hack if we want to use the loader only, permitting things like the gdb bridge to work as you were mentioning in the other comment.